internal/spice/config.go

```diff
@@ -58,9 +61,13 @@ type Config struct {
 	// items is a map from configuration key (without the "spice." prefix)
 	// to list of values for that field.
 	items map[string][]string
+
+	// shorthands is a map from shorthand to the list of arguments
+	// that that it expands to.
+	shorthands map[string][]string
 }
 
-// ConfigOptions specifies options for the [Config].
+// ConfigOptions spceifies options for the [Config].
```

Comment:
```suggestion
// ConfigOptions specifies options for the [Config].
```

Reply:
Oh, oops. Guess I undid the fix. Thanks for catching this.